src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,50 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int count = 0;
+        int[] mainDiagonal = new int[matrix.length];
```

Comment:
read checklist attentively:
- Use only one for loop to find diagonals of matrix, because the matrix is square.

Reply:
Ok I do, but may I left two loops for find min and max ?!